pathd/subdir.am

```diff
@@ -29,6 +29,10 @@ pathd_libpath_a_SOURCES = \
 	pathd/pathd.c \
 	# end
 
+clippy_scan += \
```

Comment:
Should this really be there ?

Reply:
Yes, I got an automake configure warning if its not. I noticed its in the ldpd code, and copied it from there.